vendor/assets/javascripts/koi/jquery.placeholder.js

```diff
@@ -1,104 +1,281 @@
-(function($) {
-  // @todo Document this.
-  $.extend($,{ placeholder: {
-      browser_supported: function() {
-        return this._supported !== undefined ?
-          this._supported :
-          ( this._supported = !!('placeholder' in $('<input type="text">')[0]) );
-      },
-      shim: function(opts) {
-        var config = {
-          color: '#888',
-          cls: 'placeholder',
-          selector: 'input[placeholder], textarea[placeholder]'
+/*!
+ * jQuery Placeholder Plugin v2.3.1
```

Comment:
Koi project board notes say that this is unused.
if it's unused can we delete it? or did you find a use?